docs/index.html

```diff
@@ -0,0 +1,241 @@
+          <td>10</td>
+          <td>$100,000.00</td>
+        </tr>
+        <tr>
+          <td>Quesadilla</td>
+          <td>1</td>
+          <td>$2.22</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h3 id="typography">Typography</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque dictum hendrerit velit, quis
+      ullamcorper sem congue ac. Quisque id magna rhoncus, sodales massa vel, vestibulum elit. Duis
+      ornare accumsan egestas. Proin maximus lacus interdum leo molestie convallis. Orci varius
+      natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Ut iaculis risus eu
+      felis feugiat, eu mollis neque elementum. Donec interdum, nisl id dignissim iaculis, felis dui
+      aliquet dui, non fermentum velit lectus ac quam. Class aptent taciti sociosqu ad litora
+      torquent per conubia nostra, per inceptos himenaeos.
+      <strong>This is strong,</strong> this is normal, <b>this is just bold,</b>
+      <em>and this is emphasized!</em> And heck, <a href="/">here</a>'s a link.
+    </p>
+
+    <blockquote cite="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote">
+      "The HTML blockquote Element (or HTML Block Quotation Element) indicates that the enclosed
+      text is an extended quotation. Usually, this is rendered visually by indentation (see
+      <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote#Usage_notes"
+        >Notes</a
+      >
+      for how to change it). A URL for the source of the quotation may be given using the
+      <code>cite</code> attribute, while a text representation of the source can be given using the
+      <code>&lt;cite&gt;</code> cite element."
+
+      <footer>
+        <cite>MDN, "The Block Quotation element"</cite>
+      </footer>
+    </blockquote>
+
+    <ul>
+      <li>Unordered list item 1</li>
+      <li>Unordered list item 2</li>
+      <li>Unordered list item 3</li>
+    </ul>
+    <ol>
+      <li>Ordered list item 1</li>
+      <li>Ordered list item 2</li>
+      <li>Ordered list item 3</li>
+    </ol>
+
+    <p>Addresses are also styled to be <strong>awesome</strong>!</p>
+    <address>
+      <a href="mailto:john.doe@example.com">john.doe@example.com</a><br />
+      <a href="tel:778-330-2389">778-330-2389</a><br />
+      <a href="sms:666-666-6666">666-666-6666</a><br />
+    </address>
+
+    <br />
+
+    <h1>Heading 1</h1>
+    <h2>Heading 2</h2>
+    <h3>Heading 3</h3>
+    <h4>Heading 4</h4>
+    <h5>Heading 5</h5>
+    <h6>Heading 6</h6>
+
+    <footer>
+      <a href="#">Back to top <span class="emoji">⬆</span></a>
+    </footer>
+
+    <script
+      src="https://unpkg.com/clipboard-polyfill@2.8.0/build/clipboard-polyfill.promise.js"
+      defer
+    ></script>
+    <script src="https://unpkg.com/@ungap/url-search-params@0.1.2/min.js" defer></script>
+    <script src="https://unpkg.com/vue@2.6.10/dist/vue.min.js" defer></script>
```

Comment:
I'm not a Vue expert, so I'm trusting your expertise here!

It would be nice if we could include our docs dependencies in `package.json`, but I'm not sure how to keep that separate from the actual water.css dependencies. Maybe this is best for now?

Reply:
Vue is already listed in the `devDependencies`, but you're right, we should also list the clipboard and URLSearchParams polyfills there so everything is documented.

Or do you mean that we should bundle and ship the dependencies for the docs site ourselves? That can probably be added to the gulp task.

Reply:
I added the other two deps to `devDependencies`.
As for building our own JS bundle, I think we can also do this at a later point in time.